test/karma_test.go

```diff
@@ -0,0 +1,178 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	k8sutils "github.com/mesosphere/kubeaddons-extrasteps/pkg/utils"
+	"github.com/mesosphere/kubeaddons/pkg/test"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	karmaAlertsPath    = "/ops/portal/kommander/monitoring/karma/alerts.json"
+	karmaHealthyStatus = "success"
+	karmaPodPrefix     = "kommander-kubeaddons-karma-"
+	karmaPort          = "8080"
+	ns                 = "kommander"
+)
+
+// -----------------------------------------------------------------------------
+// Karma - Tests
+// -----------------------------------------------------------------------------
+
+func karmaChecker(t *testing.T, cluster test.Cluster) test.Job {
+	return func(t *testing.T) error {
+		// find the running Karma Pod
+		karmaPod, err := findKarmaPod(cluster)
```

Comment:
nit: I would say the above comment isn't necessary considering the function name

Reply:
changed :+1: